db/src/models/challenge.rs

```diff
@@ -87,7 +87,7 @@ impl NewChallenge {
             });
         };
         Ok(Self {
-            nanoid: nanoid!(10),
+            nanoid: nanoid!(12),
```

Comment:
Will this play nice once we deploy if there are any open challenges with the old length left open?

Reply:
Yes, because this is just the call that creates them, the database doesn't care how long they are. :)